### PR TITLE
{cae}[foss/2020b] OpenFOAM v8, ParaView v5.8.1, CGAL v5.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ generoso
CORE_COUNT=16

File: easybuild/easyconfigs/p/ParaView/ParaView-5.8.1-foss-2020b-mpi.eb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+configopts = '-DPARAVIEW_INSTALL_DEVELOPMENT_FILES=ON -DPARAVIEW_BUILD_SHARED_LIBS=ON '
+configopts += '-DPARAVIEW_USE_MPI=ON '
+configopts += '-DPARAVIEW_ENABLE_FFMPEG=ON '
+configopts += '-DPARAVIEW_USE_PYTHON=ON '
+configopts += '-DPython3_ROOT_DIR=$EBROOTPYTHON '
+
+# OpenGL & Mesa
+configopts += '-DOPENGL_glu_LIBRARY=$EBROOTLIBGLU/lib/libGLU.%s ' % SHLIB_EXT
+configopts += '-DOPENGL_INCLUDE_DIR=$EBROOTMESA/include '
+configopts += '-DVTK_OPENGL_HAS_OSMESA=ON '
```
Comment (boegel) [inline easybuild/easyconfigs/p/ParaView/ParaView-5.8.1-foss-2020b-mpi.eb:47]: This may need to be dropped, cfr. discussion in #10989 and #10977...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3555.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/f10241d37486a25321dcf361ab328359 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3136.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/dc2287ab80c02af441945b001c924713 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node2649.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/627801edc54260cc8fb8c740f6f55411 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3108.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/5e149d8a2965c1c270820b4264ce4282 for a full test report.

**Comment by furstj** [comment]
Test report by @furstj
**SUCCESS**
Build succeeded for 7 out of 7 (3 easyconfigs in total)
pc-12101-fuji - Linux UNKNOWN UNKNOWN, x86_64, Intel(R) Xeon(R) CPU E3-1241 v3 @ 3.50GHz, Python 3.6.12
See https://gist.github.com/9e14b0fa34e8e998ab6a81c79f95946b for a full test report.

**Comment by Mormacill** [comment]
@boegel:
Don't forget about gnuplot dependency as mentioned in #11770.
Also check #11975 and #11974

**Comment by Mormacill** [comment]
> 
> 
> @boegel:
> Don't forget about gnuplot dependency as mentioned in #11770.
> Also check #11975 and #11974

gnuplot 5.4.1 GCCcore/10.2.0 #12051

File: easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-8-foss-2020b.eb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+dependencies = [
+    ('libreadline', '8.0'),
+    ('ncurses', '6.2'),
+    # OpenFOAM requires 64 bit METIS using 32 bit indexes (array indexes)
+    ('METIS', '5.1.0'),
+    ('SCOTCH', '6.1.0'),
+    ('CGAL', '5.2'),
+    ('ParaView', '5.8.1', '-mpi'),
+]
```
Comment (branfosj) [inline easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-8-foss-2020b.eb:31]:
```suggestion
    ('gnuplot', '5.4.1'),
]
```

Add gnuplot, as per https://github.com/easybuilders/easybuild-easyblocks/pull/2256

**Review by branfosj — CHANGES_REQUESTED** [review]

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
Test report by @branfosj
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2256
**SUCCESS**
Build succeeded for 11 out of 11 (3 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/eb3c5b0570eb49758d68511265c663f2 for a full test report.

**Comment by branfosj** [comment]
@boegelbot please test @ generoso
EB_ARGS="--include-easyblocks-from-pr 2256"

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**